lake/root.go

```diff
@@ -26,16 +26,15 @@ import (
 )
 
 var (
-	ErrPoolExists     = errors.New("pool already exists")
-	ErrPoolNotFound   = errors.New("pool not found")
+	//ErrPoolExists = errors.New("pool already exists")
+	//ErrPoolNotFound   = errors.New("pool not found")
 	ErrBranchExists   = errors.New("branch already exists")
 	ErrBranchNotFound = errors.New("branch not found")
 )
```

Comment:
The errors in `lake/pools` make these branch errors redundant, so all of this should go away.